homeassistant/components/doorbird/button.py

```diff
@@ -70,6 +70,8 @@ async def async_setup_entry(
 class DoorBirdButton(DoorBirdEntity, ButtonEntity):
     """A relay in a DoorBird device."""
 
+    entity_description: DoorbirdButtonEntityDescription
```

Comment:
Why is this needed? The `entity_description` attribute is typed in the init method.

Reply:
`entity_description` is a class attribute, and it needs to be overriden as it defaults to `ButtonEntityDescription` from `ButtonEntity`

Reply:
It is very often the case that it needs to be overriden, see:
https://github.com/home-assistant/core/blob/d1a2ce4b78ffcccd9e43f2604846b6d3fb9abc73/homeassistant/components/zwave_js/binary_sensor.py#L387-L402